src/preprocessing/passes/global_negate.cpp

```diff
@@ -27,7 +27,8 @@ namespace CVC4 {
 namespace preprocessing {
 namespace passes {
 
-Node GlobalNegate::simplify(std::vector<Node>& assertions, NodeManager* nm)
+Node GlobalNegate::simplify(const std::vector<Node>& assertions,
+                            NodeManager* nm)
```

Comment:
maybe we wanna drop having the node manager as a parameter? What comes first to mind is not that this is being done to save a call currentNM().

Reply:
Its also potentially to make it possible to eliminate calls to currentNM() in case we ever decide to make NodeManager a non-singleton.  I'd prefer keeping this for now.

Reply:
I see. OK.